inductiva/structures/scenarios/plate_linear_elastic/plate_linear_elastic.py

```diff
@@ -0,0 +1,112 @@
+# pylint: disable=unused-argument
+"""Plate linear elastic scenario."""
+
+import os
+from typing import Optional, List
+from uuid import UUID
+
+from inductiva import types, tasks
+from inductiva import structures
+from inductiva import simulation
+from inductiva import scenarios
+
+from bc_utils import BoundaryConditionsCase
+from geometry_utils import CircularHole
+from geometry_utils import EllipticalHole
+from geometry_utils import GeometricCase
+from geometry_utils import RectangularHole
+from geometry_utils import RectangularPlate
+from material_utils import IsotropicLinearElasticMaterial
+from mesh_utils import Mesh
```

Comment:
Remember that you shouldn't import classes directly. Import the submodules instead.